generator/src/googleapis/codegen/php_generator.py

```diff
@@ -229,6 +232,14 @@ class PhpLanguageModel(language_model.LanguageModel):
       'null', 'resource',
       ))
 
+  PHP_PRIMITIVE_TYPES = frozenset((
+      'bool',
+      'int',
+      'float',
+      'string',
+      'array',
```

Comment:
Wouldn't this then render as `array[]`?

Reply:
_possible_, but that seems to be handled the `dataType` property that's set here: https://github.com/googleapis/google-api-php-client-services/blob/3f5446122ce80bf4bafaf676969035a6edc452aa/generator/src/googleapis/codegen/php_generator.py#L120

Which seems like it would only get used by an array of complex types.